backend/ee/onyx/configs/app_configs.py

```diff
@@ -34,7 +34,7 @@
 
 
 # Super Users
-SUPER_USERS = json.loads(os.environ.get("SUPER_USERS", '["pablo@danswer.ai"]'))
+SUPER_USERS = json.loads(os.environ.get("SUPER_USERS", '["pablo@onyx.app"]'))
```

Comment:
LOL this seems kinda suspect XD should probably just make it empty?

Reply:
This is fair!